concepts/05 Widgets/DataGrid/20 Editing/10 User Interaction/10 Row Mode.md

```diff
@@ -2,7 +2,7 @@ In this mode a user edits data row by row. When a user clicks an *"Edit"* button
 
 ![DevExtreme HTML5 JavaScript jQuery Angular Knockout Widget DataGrid Editing Row Mode Editing State](/images/DataGrid/editing/row_mode_editing_state.png)
 
-Clicking a *"Delete"* button invokes the confirmation dialog that allows a user to cancel row deletion. Use the code below to hide this dialog.
+Clicking a *"Delete"* button invokes the confirmation dialog that allows a user to cancel row deletion. [confirmDelete](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/editing/confirmDelete.md '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/editing/#confirmDelete') option to hide this dialog:
```

Comment:
```suggestion
Clicking the *"Delete"* button invokes the confirmation dialog that allows a user to cancel row deletion. [confirmDelete](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/editing/confirmDelete.md '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/editing/#confirmDelete') option to hide this dialog:
```

Reply:
incomplete sentence:
[confirmDelete](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/editing/confirmDelete.md '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/editing/#confirmDelete') option to hide this dialog: